doc/apps/dgst.pod

```diff
@@ -14,21 +14,24 @@ B<openssl> B<dgst>
 [B<-binary>]
 [B<-r>]
 [B<-non-fips-allow>]
-[B<-out filename>]
-[B<-sign filename>]
-[B<-keyform arg>]
-[B<-passin arg>]
-[B<-verify filename>]
-[B<-prverify filename>]
-[B<-signature filename>]
-[B<-hmac key>]
+[B<-out> I<filename>]
+[B<-sign> I<filename>]
+[B<-keyform> I<arg>]
+[B<-passin> I<arg>]
+[B<-verify> I<filename>]
+[B<-prverify> I<filename>]
+[B<-signature> I<filename>]
+[B<-hmac> I<key>]
+[B<-mac> I<alg>]
+[B<-macopt> I<nm>B<:>I<v> ...]
+[B<-rand> I<files>]
 [B<-non-fips-allow>]
 [B<-fips-fingerprint>]
-[B<file...>]
+[I<file ...>]
```

Comment:
I agree, the ... is a syntactic element like "names ..." it should all be treated the same.